src/modules/dashboard/components/Incorporation/IncorporationForm/LockedProtectors/LockedProtectors.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { FormSection, InputLabel } from '~core/Fields';
+import QuestionMarkTooltip from '~core/QuestionMarkTooltip';
+import { ValuesType } from '~pages/IncorporationPage/types';
+import UserAvatar from '~core/UserAvatar';
+import UserMention from '~core/UserMention';
+import Tag from '~core/Tag';
+
+import { SignOption, VerificationStatus } from '../constants';
+
+import styles from './LockedProtectors.css';
+
+export const MSG = defineMessages({
+  protectors: {
+    id: `dashboard.Incorporation.IncorporationForm.LockedProtectors.protectors`,
+    defaultMessage: 'Protectors',
+  },
+  protectorsTooltip: {
+    id: `dashboard.Incorporation.IncorporationForm.LockedProtectors.protectorsTooltip`,
+    defaultMessage: `A Protector's role in a DAO legal corporation is to ratify the decisions of the DAO. Their purpose is to act on behalf of the DAO and handle legal the required administration. Learn more`,
+  },
+  unverified: {
+    id: `dashboard.Incorporation.IncorporationForm.LockedProtectors.unverified`,
+    defaultMessage: 'Unverified',
+  },
+  verified: {
+    id: `dashboard.Incorporation.IncorporationForm.LockedProtectors.verified`,
+    defaultMessage: 'Verified',
+  },
+  mainContact: {
+    id: `dashboard.Incorporation.IncorporationForm.LockedProtectors.mainContact`,
+    defaultMessage: 'Main contact',
+  },
+  signing: {
+    id: `dashboard.Incorporation.IncorporationForm.LockedProtectors.signing`,
+    defaultMessage: 'Signing',
+  },
+  mainContactTooltip: {
+    id: `dashboard.Incorporation.IncorporationForm.LockedProtectors.mainContactTooltip`,
+    defaultMessage: `The main contact is required during the incorporation process and is also required to use their delivery address details for the registration.`,
```

Comment:
Please change to: "The main contact is required during the incorporation process for administration purposes."